### PR TITLE
feat(atomicradio): add atomicradio presence

### PR DESCRIPTION
**Pull request by VocalZero**
## Description 

Added a presence for https://atomic.radio

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

<br>

</details>

### CODE REVIEW

File: websites/A/atomicradio/presence.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "904084297831571518",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
```
Comment (theusaf) [inline websites/A/atomicradio/presence.ts:8]:
```suggestion
			largeImageKey: "https://i.imgur.com/16lVdWy.png",
```

File: websites/A/atomicradio/presence.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			smallImageKey: "search",
```
Comment (theusaf) [inline websites/A/atomicradio/presence.ts:9]: Use image links or [provided assets](https://github.com/PreMiD/Presences/blob/c598cf31f004d621445e57a67b41c158b291fd2f/%40types/premid/index.d.ts#L78-L96) over Discord asset names.
```suggestion
			smallImageKey: Assets.Search,
```

File: websites/A/atomicradio/presence.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+		).textContent;
+		presenceData.largeImageKey = player.querySelector(
+			"div.container > #artwork"
+		).textContent;
+		presenceData.smallImageKey =
+			player.querySelector<HTMLButtonElement>(
+				"div.container > div.center-controls > button"
+			).id === "play-button"
+				? "pause"
+				: "play";
```
Comment (theusaf) [inline websites/A/atomicradio/presence.ts:26]:
```suggestion
				? Assets.Pause
				: Assets.Play;
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/atomicradio/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+		"name": "VocalZero",
+		"id": "223891083724193792"
+	},
+	"service": "atomicradio",
+	"description": {
+		"en": "The future of radio is here. Experience 12 Spaces with non-stop music you'll love."
+	},
+	"url": "atomic.radio",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/16lVdWy.png",
```
Comment (DarkVillager) [inline websites/A/atomicradio/metadata.json:13]: Logo's must be 512x512 as per our [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

File: websites/A/atomicradio/presence.ts [hunk 5]
```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "904084297831571518",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/16lVdWy.png",
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:8]: Ditto (Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).)

File: websites/A/atomicradio/presence.ts [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+			startTimestamp: browsingTimestamp,
+		},
+		player = document.querySelector<HTMLDivElement>("div.player");
+
+	if (player) {
+		const spaceId = player.querySelector(
+			"div.container > #spaceId"
+		).textContent;
+		presenceData.largeImageKey = player.querySelector(
+			"div.container > #artwork"
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:19]: Try to shorten selectors where possible like here:
```suggestion
			"#artwork"
```

File: websites/A/atomicradio/presence.ts [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+		).textContent;
+		presenceData.smallImageKey =
+			player.querySelector<HTMLButtonElement>(
+				"div.container > div.center-controls > button"
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:23]: Ditto. (Try to shorten selectors where possible like here:)
```suggestion
				'[id*="-button"]'
```

File: websites/A/atomicradio/presence.ts [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+			).id === "play-button"
+				? Assets.Pause
+				: Assets.Play;
+		presenceData.details = player.querySelector<HTMLAnchorElement>(
+			"div.container > a > div.track-info > a.track-title"
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:28]: Ditto. (Try to shorten selectors where possible like here:)
```suggestion
			"a.track-title"
```

File: websites/A/atomicradio/presence.ts [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+		).textContent;
+		presenceData.state = player.querySelector<HTMLDivElement>(
+			"div.container > a > div.track-info > div.track-artist"
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:31]: Ditto. (Try to shorten selectors where possible like here:)
```suggestion
			"div.track-artist"
```

File: websites/A/atomicradio/presence.ts [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/16lVdWy.png",
+			smallImageKey: Assets.Search,
+			startTimestamp: browsingTimestamp,
+		},
+		player = document.querySelector<HTMLDivElement>("div.player");
+
+	if (player) {
+		const spaceId = player.querySelector(
+			"div.container > #spaceId"
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:16]: Ditto. (Try to shorten selectors where possible like here:)
```suggestion
			" #spaceId"
```

**Review by Bas950 — APPROVED** [review]

File: websites/A/atomicradio/presence.ts [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/vPI68Ny.png",
+			smallImageKey: Assets.Search,
+			startTimestamp: browsingTimestamp,
+		},
+		player = document.querySelector<HTMLDivElement>("div.player");
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:12]:
```suggestion
		player = document.querySelector<HTMLDivElement>("div.player"),
		{pathname} = document.location;
```

File: websites/A/atomicradio/presence.ts [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+			presenceData.details = "Viewing contributors";
+		else if (document.location.pathname.includes("/partners"))
+			presenceData.details = "Viewing partners";
+		else if (document.location.pathname.includes("/legal/imprint"))
+			presenceData.details = "Viewing imprint";
+		else if (document.location.pathname.includes("/legal/privacy"))
+			presenceData.details = "Viewing privacy";
+		else if (document.location.pathname.includes("/account"))
+			presenceData.details = "Viewing account";
+	}
```
Comment (DarkVillager) [inline websites/A/atomicradio/presence.ts:53]:
```suggestion
		if (pathname.includes("/statistics"))
			presenceData.details = "Viewing statistics";
		else if (pathname.includes("/streams"))
			presenceData.details = "Viewing stream urls";
		else if (pathname.includes("/about"))
			presenceData.details = "Viewing about us";
		else if (pathname.includes("/apply"))
			presenceData.details = "Viewing apply";
		else if (pathname.includes("/contributors"))
			presenceData.details = "Viewing contributors";
		else if (pathname.includes("/partners"))
			presenceData.details = "Viewing partners";
		else if (pathname.includes("/legal/imprint"))
			presenceData.details = "Viewing imprint";
		else if (pathname.includes("/legal/privacy"))
			presenceData.details = "Viewing privacy";
		else if (pathname.includes("/account"))
			presenceData.details = "Viewing account";
	}
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**